RecoTracker/MeasurementDet/plugins/MeasurementTrackerEventProducer.cc

```diff
@@ -189,7 +189,8 @@ void MeasurementTrackerEventProducer::updatePixels(const edm::Event& event,
           // PixelFEDChannelCollection addresses the ROCs by their 'idInDetUnit' (from 0 to 15), ROCs also know their on 'idInDetUnit',
           // however the cabling map uses a numbering [1,numberOfROCs], see sipixelobjects::PixelFEDLink::roc(unsigned int id), not necessarily sorted in the same direction.
           // PixelFEDChannelCollection MUST be filled such that ch.roc_first (ch.roc_last) correspond to the lowest (highest) 'idInDetUnit' in the channel
-          for (path.roc = 1; path.roc <= (ch.roc_last - ch.roc_first) + 1; path.roc++) {
+          assert(ch.roc_last >= ch.roc_first);
```

Comment:
Shouldn't this be guaranteed at the unpacker level?

Reply:
It should... but in case it's not the case (e.g. while debugging) better to have an explicit assert than a loop that takes a few minutes per module:-(

Reply:
Ok. Thanks, I just wanted to make sure the behaviour did not change for some deeper reason.